src/SRS/srs_problem.c

```diff
@@ -478,7 +480,7 @@ int SRSgetobjval(SRS_PROBLEM * problem_srs, double * objVal) {
 	if (problem_srs->maximize)
 		(*objVal) *= -1.;
 
-    *objVal += problem_srs->objective_offset;
+    *objVal += problem_mps->objective_offset;
```

Comment:
this line should be before line 480 to take into account the min/max

Reply:
As discussed, not really
```
argmax {a.x + b} = argmax {a.x} + b = -argmin {-a.x} + b
```